staging/src/k8s.io/client-go/go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	golang.org/x/time v0.0.0-20200630173020-3af7569d3a1e
 	k8s.io/api v0.0.0
 	k8s.io/apimachinery v0.0.0
+	k8s.io/klog v1.0.0
 	k8s.io/klog/v2 v2.4.0
```

Comment:
Doesn't seem right to have both v1 and v2 in this list?

Reply:
Indeed! At It was also unexpected.
It should be fixed in 961fa25.